src/program/operations/delete/index.js

```diff
@@ -29,6 +29,7 @@ const shouldDeleteScript = scriptName => {
         safeExit();
       } else {
         GlobalConfig.removeScript(scriptName);
```

Comment:
Technically, we don't need to remove the script here since saving the script only saves script names and so long as we remove the script name it will remove the yaml file when the config is saved but it's not bad to be defensive